api/pom.xml

```diff
@@ -20,14 +20,14 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>org.eclipse.ee4j.xml.registry-api</groupId>
-        <artifactId>xml.registry-api-parent</artifactId>
-        <version>1.0.9-SNAPSHOT</version>
+        <groupId>jakarta.xml.registries-api</groupId>
```

Comment:
Nope, don't change these coordinates.  It's just the name of the project, not the coordinates.  That would mess up existing users.